### PR TITLE
Milestone 3: Contact form

### PR DESCRIPTION
**Pull request by mavericks-db**
## Portfolio: Add a contact form and finish the mobile version

Changes included:
-add a contact form
-finish the mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -182,42 +182,63 @@ <h2>About Me</h2>
...
         </li>
       </ul>
     </div>
   </section>
+
+  <!-- Contact Form -->
+  <section class="contact-form">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form action="https://formspree.io/f/xwkyqzow" method="post">
+      <input type="text" name="name" placeholder="Full name" required>
```
Comment (Deepakdanger) [inline index.html:226]: - [ ] **Required** Name input has maxlength="30" attribute. Lets add this.

File: index.html [hunk 2]
```diff
@@ -182,42 +182,63 @@ <h2>About Me</h2>
...
+      <input type="email" name="email" placeholder="Email address" required>
+      <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required></textarea>
```
Comment (Deepakdanger) [inline index.html:228]: - [ ] **Required** Message input has `maxlength="500"` attribute. Let's add this.

File: index.html [hunk 3]
```diff
@@ -182,42 +182,63 @@ <h2>About Me</h2>
...
+      <button type="submit">Get in touch</button>
+    </form>
+    <div class="iconLinks">
+      <img src="./assets/twitter.svg" alt="twitter">
+      <img src="./assets/linkedin.svg" alt="linkedin">
+      <img src="./assets/medium.svg" alt="medium">
+      <img src="./assets/github.svg" alt="github">
+      <img src="./assets/angellist.svg" alt="angellist">
+    </div>
```
Comment (Deepakdanger) [inline index.html:237]: - [ ] All this icon should be in `ul` & `li` tag,  & each `li` tag be inside `a` tag. Let's fix this

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @mavericks-db, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Rustamxon7 — APPROVED** [review]
## Hello @mavericks-db 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**